DANS-KNAW/easy-sword2-dans-examples
# Fetch example

fixes EASY-1099
#### When applied it will
- add a new example bag with fetch.txt
#### What needs to be done by @rvanheest before this PR can be merged?
- [x] test what goes wrong when this bag would have an empty file in its `fetch.txt`. Sometimes the file cannot be downloaded from [GitHub](https://github.com/DANS-KNAW/easy-sword2-dans-examples/tree/master/src/main/resources/fetch-files) and instead it gives a 503 Service Not Available error. Does this also happen with non-empty files? Should we include a 'wait-and-retry' policy in easy-sword2 for this?
- [x] test what goes wrong when this bag contains an empty file that is successfully fetched from GitHub. In these cases we get a 500 error from Fedora. Does this have to do with the file being empty (e.g. 0 bytes)? Should we do anything about this?
- [x] review the additions to the README. Are these still valid?
#### Where should the reviewer @DANS-KNAW/easy start?

This PR contains a new bag with a `fetch.txt`. Happy reading. Consider it all to be random sequences of characters 😉 
#### How should this be manually tested?
1. deploy https://github.com/DANS-KNAW/easy-sword2/pull/64 on deasy
2. follow the instructions in the README
3. you should be able to completely submit a bag into easy and view it in webapp
#### related pull requests on github

| repo | PR | note |
| --- | --- | --- |
| easy-sword2 | [PR#64](https://github.com/DANS-KNAW/easy-sword2/pull/64) | the example in this PR uses PR#64. merge/deploy this one first! |
